lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -387,12 +475,15 @@ def trace_charm(
     method calls on instances of this class.
 
     Usage:
+    >>> import scenario
```

Comment:
I wonder if this should be part of the default path since `state` is optional. Will `charm_tracing` work without importing scenario and snapshot lib? If so, we might want to split the documentation into two versions - basic as a minimal instrumentation and extended with `state` and `extra_types`.

Reply:
good idea. Will put that in the module docstring instead.